### ISSUE TITLE
synctl install error:Unrecognized command or argument 'kubernetes'

### ISSUE THREAD

**Issue by Yanghaihai1020** [serverlessworkflow/synapse#329]
**What happened**:
when I tried `synctl system install kubernetes`,it shows: Unrecognized command or argument 'kubernetes'. 
```
root@node7:/usr/yn/workflow# synctl system install kubernetes
Required command was not provided.
Unrecognized command or argument 'kubernetes'.

Description:
  Installs Synapse

Usage:
  synctl system install [command] [options]

Options:
  -?, -h, --help  Show help and usage information

Commands:
  native  Installs Synapse on the current OS
  docker  Installs Synapse on Docker

```
**What you expected to happen**:
synapse successfully install on kubernetes.
**How to reproduce it**:

**Anything else we need to know?**:

**Environment**:
Ubuntu v22.04
kubernetes: v1.22.3

**Comment by cdavernas** [comment]
@Yanghaihai1020 Thanks for reporting, I'll look into it!

**Comment by cdavernas** [comment]
Closed as irrelevent in upcoming 1.0.0
